### PR TITLE
Change: Add placeholder and tooltip to clarify IPAM address format

### PR DESCRIPTION
**Pull request by Jskobos**
## Description

Per Chris's suggestion, be a little clearer about the required format for IPAM addresses.

### CODE REVIEW

**Comment by johnwcallahan** [comment]
Looks good; I pushed up a few changes, mostly to fix the error message positioning:

Screen Shot 2021-04-12 at 4 44 58 PM

Also changed the color of the tooltip in TextField which broke the StackScript page, so I made an adjustment in ImageSelect to fix that.

Since these are global changes, I'd really love a review from you @Jskobos and @tiffwong.

File: packages/manager/src/components/TextField.tsx [hunk 1]
```diff
@@ -395,21 +396,21 @@ class LinodeTextField extends React.PureComponent<CombinedProps> {
...
-                [classes.absolute]: editable || hasAbsoluteError,
-              })}
-              data-qa-textfield-error-text={this.props.label}
-              role="alert"
-            >
-              {errorText}
-            </FormHelperText>
-          )}
         </div>
+        {errorText && (
```
Comment (johnwcallahan) [inline packages/manager/src/components/TextField.tsx:400]: I moved the end of this div before the error message to fix the positioning.

File: packages/manager/src/features/Images/ImageSelect.tsx [hunk 2]
```diff
@@ -15,8 +15,9 @@ const useStyles = makeStyles((theme: Theme) => ({
     width: '100%',
   },
   icon: {
-    marginTop: theme.spacing(2) + 14,
-    marginLeft: -theme.spacing(1),
+    marginTop: 30,
+    marginLeft: -20,
+    color: theme.cmrTextColors.tableHeader,
```
Comment (johnwcallahan) [inline packages/manager/src/features/Images/ImageSelect.tsx:20]: This is to fix the color on the StackScript Create page, which became inconsistent after the change in TextField. I fixed the positioning while I was here.

**Review by johnwcallahan — APPROVED** [review]

**Review by tiffwong — APPROVED** [review]
Everything lgtm!

**Review by Jskobos — COMMENTED** [review]
I can't find anywhere that the Textfield changes have broken. There are some spacing issues but they're in production too:

Screen Shot 2021-04-13 at 9 40 06 AM

**Pull request merged**